baselines/doc/source/index.rst

```diff
@@ -19,30 +19,32 @@ The Flower Community is growing quickly - we're a friendly group of researchers,
 Flower Baselines
 ----------------
 
-Flower Baselines are a collection of organised scripts used to reproduce results from well-known publications or benchmarks. You can check which baselines already exist and/or contribute your own baseline.
+Flower Baselines are a collection of organised directories used to reproduce results from well-known publications or benchmarks. You can check which baselines already exist and/or contribute your own baseline.
 
 .. BASELINES_TABLE_ANCHOR
 
-Tutorials
-~~~~~~~~~
 
-A learning-oriented series of tutorials, the best place to start.
+How-to guides
+~~~~~~~~~~~~~
+
+Problem-oriented how-to guides show step-by-step how to achieve a specific goal.
```

Comment:
It's ok to move our two docs pages to the `How-To` section, but let's keep the order of Tutorial / How-To / Explanation / Reference the same as it is for other projects

Reply:
ok